chart/templates/nginx.yaml

```
@@ -21,6 +21,11 @@ data:
             {{- if .Values.customGateway | default false}}
             *.{{ .Values.customGateway.domainName }} backend_custom_gateway;
             {{- end }}
+            {{- if .Values.passthrough.enabled | default false}}
+            {{- range .Values.passthrough.upstreams }}
+            {{ . }}.{{ $.Values.domain }} backend_{{ . }};
```

This is a bit of a nit but I think we can simplify and just use a single backend (see other comment as well):
```suggestion
            {{ . }}.{{ $.Values.domain }} backend_passthrough;
```